src/cdk-experimental/menu/menu.md

```diff
@@ -0,0 +1,167 @@
+The `@angular/cdk-experimental/menu` module provides you with a way to easily create complex menu interfaces with support for:
+
+- Standalone Menus
+- Menubars
+- Context Menus
+- Inline Menus
+- MenuItems
+- RadioButtonItems
+- CheckboxItems
+
+`@angular/cdk-experimental/menu` can also intelligently predict if a user is attempting to navigate to an open submenu and prevent premature closeouts.
+
```

Comment:
I would move this to its own section named something like "Smart menu aim" that outlines what it does (potentially with a diagram) and a very brief explanation of how it works.